server/graphql/repack/lib.rs

```diff
@@ -1,6 +1,23 @@
 pub mod mutations;
+pub mod query;
 use async_graphql::*;
 use mutations::{insert_repack, InsertRepackInput, InsertResponse};
+use query::{get_repack, RepackResponse};
+
+#[derive(Default, Clone)]
+pub struct RepackQueries;
+
+#[Object]
+impl RepackQueries {
+    pub async fn repack(
+        &self,
+        ctx: &Context<'_>,
+        store_id: String,
+        invoice_id: String,
```

Comment:
Insert response returns the invoice_id, so should be able to use that to print out the repack...

Reply:
I think repack_id would be good here. in query many we should just be able to query repacks by stock_line_id though, I think we can still use invoice repository but add stock_line_id to it, with inner query:

Something like this: https://github.com/openmsupply/open-msupply/blob/9194d8e05ceb5bd33b80baf2e01f821cf3b05e3c/server/repository/src/db_diesel/master_list.rs#L80-L90

Reply:
Query one by repack_id instead of invoice_id? 🤔 

https://github.com/openmsupply/open-msupply/pull/1825 <- Query many 😉